includes/Admin/Editor.php

```diff
@@ -279,15 +284,17 @@ public function admin_enqueue_scripts( $hook ) {
 			true
 		);
 
-		$script_dependencies = [ Tracking::SCRIPT_HANDLE, 'postbox', self::AMP_VALIDATOR_SCRIPT_HANDLE ];
-
-		$this->assets->enqueue_script_asset( self::SCRIPT_HANDLE, $script_dependencies, false );
+		wp_enqueue_script( self::SCRIPT_HANDLE );
 		$this->assets->enqueue_style_asset( self::SCRIPT_HANDLE, [ $this->google_fonts::SCRIPT_HANDLE ] );
 
 		wp_localize_script(
 			self::SCRIPT_HANDLE,
-			'webStoriesEditorSettings',
-			$this->get_editor_settings()
+			'webStories',
+			[
+				'publicPath' => $this->assets->get_base_url( 'assets/js/' ), // Required before the editor script is enqueued.
+				'localeData' => $this->assets->get_translations( self::SCRIPT_HANDLE ), // Required for i18n setLocaleData.
+				'metaBoxes'  => $this->meta_boxes->get_meta_boxes_per_location(), // Because meta boxes are not available inside edit-story.php.
```

Comment:
Odd. So the `$wp_meta_boxes` global is `null`?

Reply:
Yes, but thanks for pointing that out. I think its because we are trying to get it before `meta-boxes.php` is included. Let me try to move after the file inclusion. 

https://github.com/google/web-stories-wp/blob/444e2cdf625a8c45aa3348fa071f4c0ba175f98d/includes/templates/admin/edit-story.php#L154-L173

Reply:
Yes, that was the reason, done in https://github.com/google/web-stories-wp/pull/9775/commits/1a8db57d898caa79b448c9f1fa18438cf5e64f4c